.github/workflows/connect-dev-release.yml

```diff
@@ -0,0 +1,57 @@
+name: "[Build/Test] connect-explorer"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  pull_request:
+    paths:
+      - "packages/connect*/**"
+
+env:
+  DEV_SERVER_URL: "https://dev.suite.sldev.cz"
+
+jobs:
+  build-connect-explorer:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          lfs: true
+
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          role-to-assume: arn:aws:iam::538326561891:role/gh_actions_trezor_suite_dev_deploy
+          aws-region: eu-central-1
+
+      - name: Extract branch name
+        run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
+        id: extract_branch
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Build connect-explorer
+        env:
+          ASSET_PREFIX: /connect/${{ steps.extract_branch.outputs.branch }}
+          CI_COMMIT_REF_NAME: ${{ steps.extract_branch.outputs.branch }}
+          # TODO: maybe we don't want to build it for production for every branch
+          NODE_ENV: "production"
+          name: $CI_COMMIT_REF_NAME
+          url: https://dev.suite.sldev.cz/connect/$CI_COMMIT_REF_NAME
+        run: |
+          yarn install
+          yarn build:libs
+          yarn workspace @trezor/connect-explorer build
+
+      # this step should upload build result to s3 bucket dev.suite.sldev.cz using awscli
+      - name: Upload suite-web to dev.suite.sldev.cz
```

Comment:
This should be probably. 
`Upload connect-explorer to dev.suite.sldev.cz`

Reply:
Done